open-apparel-registry/open-apparel-registry
# Add/update parent company dropdown

## Overview

Updates the parent company search to use a dropdown, and changes the existing parent company dropdowns on the claim / claimed forms from using all contributors to just those which have been previously used as a parent company.

Connects #1746 

## Demo

![Screenshot 2022-07-20 181536](https://user-images.githubusercontent.com/4432106/180240557-f67899bc-b10f-494b-950a-5f6a790e0c5c.png)
![Screenshot 2022-07-20 182751](https://user-images.githubusercontent.com/4432106/180240559-007bf2c9-3dc7-4bdc-ad6c-46a0f9b7566d.png)
![Screenshot 2022-07-20 182954](https://user-images.githubusercontent.com/4432106/180240560-0581a476-df76-4b3b-b3a4-f50ee8890f55.png)

## Testing Instructions

* `scripts/server`
* Search, claiming a facility & editing claimed facility details should all function even with no parent companies (as you'll have after using `scripts/resetdb`)
* Add this list, and then process it using `batch_process`: 
[1598_Columbia_5_19_2022.csv](https://github.com/open-apparel-registry/open-apparel-registry/files/9160227/1598_Columbia_5_19_2022.csv)
* Search, claiming a facility & editing claimed facility details should now show a list of parent companies in the dropdown

## Checklist
